### ISSUE TITLE
ivy.gradient

### ISSUE THREAD

**Issue by Supremolink81** [unifyai/ivy#10841]
**Why should this be implemented?**
Every other function so far implemented using the jax.numpy namespace has an ivy.func_name call, and no other functions in the Ivy framework seem to handle gradients of ndarrays (only gradients of function objects).

**Links to native framework implementations**
- [jax.numpy.gradient documentation (includes source)](https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.gradient.html#jax.numpy.gradient)

**Comment by Supremolink81** [comment]
Never mind, it was my interpreter being weird.
